src/transactions/instances/transactionPayload.ts

```diff
@@ -404,9 +405,10 @@ export class MultisigTransactionPayload {
     this.transaction_payload.serialize(serializer);
   }
 
-  static deserialize(deserializer: Deserializer): MultisigTransactionPayload {
-    // This is the enum value indicating which type of payload the multi-sig transaction contains.
+  static deserialize(deserializer: Deserializer): MultiSigTransactionPayload {
+    // TODO: Support other types of payload beside EntryFunction.
+    // This is the enum value indicating which type of payload the multisig tx contains.
     deserializer.deserializeUleb128AsU32();
-    return new MultisigTransactionPayload(EntryFunction.deserialize(deserializer));
+    return new MultiSigTransactionPayload(EntryFunction.deserialize(deserializer));
```

Comment:
This has to be done on the Aptos Move framework level.  There is no way to support non-entry functions afaik at the moment.  cc @movekevin

Reply:
This is a legacy comment https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk/src/aptos_types/transaction.ts#L228